### ISSUE TITLE
Allow any branch or tag to be deployed

### ISSUE THREAD

**Issue by georgjaehnig** [trovu/trovu#172]
e.g. the branch `142-show-reload-option-in-ui` could be deployed at `142-show-reload-option-in-ui.trovu.net`.

Or the tag `1.0.3` at `1.0.3.trovu.net`.
 
This way, branches could be tested easily on other devices before merged to master, or any dev. 

Also deployed tags could be fixed versions of Trovu. Users who dislike unexpected keyword changes could "freeze" their version by e.g. using trovu as `https://1.0.3.trovu.net/process/?query=g+berlin` and could be sure that all shortcuts remain the same.

Only when they like, they could manually upgrade to a higher version, and check the diffs of `/data to see which shortcuts got updated.

Depends on #127.

**Comment by georgjaehnig** [comment]
Dismissing this idea for now, as we don't want users to be stuck using old versions. There is also `deprecated:` dealing with outdated shortcuts.

If a user *really* wants to keep using a certain version, they can clone and run Trovu on their localhost.

And for development: So far, we can deploy one dev branch.

Closing as wontfix for now.
